Tangem/Modules/Welcome/WelcomeViewModel.swift

```diff
@@ -104,6 +78,36 @@ class WelcomeViewModel: ObservableObject {
     func onDisappear() {
         navigationBarHidden = false
     }
+    
+    private func scanCard() {
+        isScanningCard = true
+        
+        userWalletRepository.unlock(with: .card(userWallet: nil)) { [weak self] result in
+            self?.isScanningCard = false
+
+            guard
+                let self,
+                let result
+            else {
+                return
+            }
```

Comment:
может все таки в одну строку )
```suggestion
            guard let self, let result else {
                return
            }
```

Reply:
Сделал